### PR TITLE
Adding argo plugin

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ itself) to look for plugins.
...
 | alp                           | [asdf-community/asdf-alp](https://github.com/asdf-community/asdf-alp)                             | [![Main workflow](https://github.com/asdf-community/asdf-alp/workflows/Main%20workflow/badge.svg)](https://github.com/asdf-community/asdf-alp/actions)                                 |
 | aria2                         | [asdf-community/asdf-aria2](https://github.com/asdf-community/asdf-aria2)                         | [![Build Status](https://github.com/asdf-community/asdf-aria2/workflows/Main%20workflow/badge.svg)](https://github.com/asdf-community/asdf-aria2/actions)                              |
+| argo                          | [sudermanjr/argo](https://github.com/asdf-argo)                                                   | [![Build Status](https://github.com/sudermanjr/asdf-argo/workflows/master/badge.svg?branch=master)](https://github.com/sudermanjr/asdf-argo/actions)                                   |
```
Comment (imbsky) [inline README.md]: It should be like this.
`[sudermanjr/argo](https://github.com/sudermanjr/asdf-argo)`

**Comment by imbsky** [comment]
The code looks good, I can merge this after you fix the README.

**Comment by sudermanjr** [comment]
Fixed that issue and rebased.

**Comment by imbsky** [comment]
Great! Thank you!

**Comment by imbsky** [comment]
Thanks @sudermanjr for your contribution!

**Pull request merged**